experiments/modern/src/runtime/System.js

```diff
@@ -45,6 +45,23 @@ class System extends MakiObject {
   messageBox(message, msgtitle, flag, notanymoreId) {
     console.log({ message, msgtitle, flag, notanymoreId });
   }
+  integerToString(value) {
+    return value.toString();
+  }
+  getPrivateInt(section, item, defvalue) {
+    return defvalue;
+  }
+  setPrivateInt(section, item, defvalue) {
```

Comment:
I suppose these will need to get lower cased at some point?

Reply:
Yea, all the new methods need to be lowercased hah. Was working on this and the other PR in parallel, and didn't wanna have all the same code in this one. I'll rebase and get everything lowercase now though :D